games-action/heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of The Alien is the sequel for the undoubtably the best adventure of all times,\ „Another World“, developed back in 1994 by Interplay exclusively\ for the Sega CD console. This is an open-source project\ which aims to bring this highly-addictive game to all modern\ consoles and computer platforms."
```

Comment:
You're missing the line-breaks after the " \"... :)
I suggest building the package once more with your changes, open it in HaikuDepot and see if the description looks good.

Reply:
![sc](https://user-images.githubusercontent.com/34230960/34322166-02fcd72e-e829-11e7-97d8-ec25e0adecf8.PNG)

Nice :)